services/core-api/app/api/mines/permits/permit/models/permit.py

```diff
@@ -12,10 +12,14 @@
 from app.api.parties.party_appt.models.mine_party_appt import MinePartyAppointment
 
 from app.api.utils.models_mixins import AuditMixin, Base
+from app.api.constants import *
 
 
 class Permit(AuditMixin, Base):
     __tablename__ = 'permit'
+    _edit_groups = [PERMIT_EDIT_GROUP, MINE_EDIT_GROUP]
```

Comment:
I don't think we want permits to be in the mine edit group.

Reply:
changed